### ISSUE TITLE
Incorrect role is defined for control ‘Close’ icon button, present on 'Page' Popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#96]
User Experience:

User would find difficulty to identify the controls if name and role are not defined correctly for controls as they won't get the context of that control and also they won't be able to perform functionality  relevant to that control. 

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Pages button.

5. Navigate to the various controls of the "Pages" window.

6. Open Developer tool using F12 key and navigate to ‘Close’ icon button.

7. Verify whether role is defined correct or not for control.

 

Actual Result:

Incorrect role is defined for control ‘Close’ icon button and control name also not defined, role is defined as “Link” but  its behavior look like ‘button’, screen reader announcing role as link and name as  ‘X’, it will be confusing for screen reader users.

 

Expected Result:

Role should be define as ‘button’ and name as ‘Close’ control ‘Close’ button, so that user can get correct information about control role and name.

 

Note:

This issue also exists with Chrome.

Similar issue throughout the application

 

MAS Reference:

MAS 4.1.2 - Name, Role, Value

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by azaslonov** [comment]
Fixed.
